descope/sdk/mgmt.go

```diff
@@ -35,7 +35,7 @@ type Tenant interface {
 	// Delete an existing tenant.
 	//
 	// IMPORTANT: This action is irreversible. Use carefully.
-	Delete(ctx context.Context, id string) error
+	Delete(ctx context.Context, id string, cascade bool) error
```

Comment:
Add here also a comment about the attribute

Reply:
You forgot this.

Reply:
done